packages/app/package.json

```diff
@@ -9,7 +9,7 @@
     "lint": "next lint"
```

Comment:
Something is off between our package-lock jsons that it keeps flip flopping - what version of node are you on?